src/Components/hero.html

```diff
@@ -0,0 +1,23 @@
+<section id="main " class="grid grid-cols-2 mt-[50px]">
+
+    <div class="text-white text-[60px] font-ralewayBold">
+
+<p>Win-Win:</p>
+<p class="flex gap-5"> <span>Europas</span> <span  class="text-primary">Digitalisierung </span></p>
+<p>profitiert von Afrikas Talenten</p>
+
+  
```

Comment:
Here there is a missing paragraph
and this can be change to heading ex: h1...h6 for accessibility